qemu/seabios_iommu.py

```diff
@@ -0,0 +1,111 @@
+import logging
+
+from virttest import error_context
+from virttest import utils_misc
+from virttest import utils_test
+from virttest import utils_disk
+from provider.storage_benchmark import generate_instance
+
+
+def check_data_disks(test, params, env, vm, session):
+    """
+    Check guest data disks (except image1)
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    :param vm: VM object
+    :param session: VM session
+    """
+    image_list = params.objects("images")
+    del image_list[0]
+    image_num = len(image_list)
+
+    error_context.context("Check data disks in monitor!", logging.info)
+    monitor_info_block = vm.monitor.info_block(False)
+    blocks = ','.join(monitor_info_block.keys())
+    for image in image_list:
+        if image not in blocks:
```

Comment:
why not directly check `if image not in monitor_info_block`?

Reply:
'blocks' is useful in test fail info. Thanks!